Tasks/Common/azure-arm-rest/azure-app-service.ts

```diff
@@ -0,0 +1,185 @@
+import msRestAzure = require('./azure-arm-common');
+import webClient = require('./webClient');
+import azureServiceClient = require('./AzureServiceClient');
+import tl = require('vsts-task-lib/task');
+import Q = require('q');
```

Comment:
can we follow the import * as pattern everywhere.

Reply:
sure.